temporalio/ui-server
# Allow disabling UI and OpenAPI

## What was changed


Adds config options to enable/disable serving UI and Open API

## Why?


Allows to only serve the HTTP API without UI and/or OpenAPI

## Checklist


1. Closes 

2. How was this tested:


`docker run -e TEMPORAL_ENABLE_UI=false -e TEMPORAL_ENABLE_OPENAPI=false --network host temporaliotest/ui`

- open http://localhost:8080/ . Should result in 404
- open http://localhost:8080/openapi/ . Should result in 404
- open http://localhost:8080/api/v1/cluster . Should output cluster info

![image](https://user-images.githubusercontent.com/11838981/139140341-17d667b8-2429-4243-8327-f12d1caada11.png)

3. Any docs updates needed?


Updated docker README.md
